spec/models/embedded_ansible_worker_spec.rb

```diff
@@ -0,0 +1,138 @@
+describe EmbeddedAnsibleWorker do
+  subject { FactoryGirl.create(:embedded_ansible_worker) }
+
+  context "ObjectManagement concern" do
+    let(:provider)       { FactoryGirl.create(:provider_embedded_ansible) }
+    let(:api_connection) { double("AnsibleAPIConnection", :api => tower_api) }
+    let(:tower_api) do
+      methods = {
+        :organizations => org_collection,
+        :credentials   => cred_collection,
+        :inventories   => inv_collection,
+        :hosts         => host_collection,
+        :job_templates => job_templ_collection,
+        :projects      => proj_collection
+
+      }
+      double("TowerAPI", methods)
+    end
+
+    let(:org_collection)       { double("AnsibleOrgCollection", :all => [org_resource]) }
+    let(:cred_collection)      { double("AnsibleCredCollection", :all => [cred_resource]) }
+    let(:inv_collection)       { double("AnsibleInvCollection", :all => [inv_resource]) }
+    let(:host_collection)      { double("AnsibleHostCollection", :all => [host_resource]) }
+    let(:job_templ_collection) { double("AnsibleJobTemplCollection", :all => [job_templ_resource]) }
+    let(:proj_collection)      { double("AnsibleProjCollection", :all => [proj_resource]) }
+
+    let(:org_resource)         { double("AnsibleOrgResource", :id => 12) }
+    let(:cred_resource)        { double("AnsibleCredResource", :id => 13) }
+    let(:inv_resource)         { double("AnsibleInvResource", :id => 14) }
+    let(:host_resource)        { double("AnsibleHostResource", :id => 15) }
+    let(:job_templ_resource)   { double("AnsibleJobTemplResource", :id => 16) }
+    let(:proj_resource)        { double("AnsibleProjResource", :id => 17) }
+
+    describe "#ensure_initial_objects" do
```

Comment:
Interesting, I agree for contexts, but I always use a `describe` for the method under test regardless of how many tests I'm writing.